kimchi/src/verifier.rs

```diff
@@ -313,10 +312,16 @@ where
             })
             .collect();
 
-        let evals = vec![
-            self.evals.z.combine(powers_of_eval_points_for_chunks[0]),
-            self.evals.zw.combine(powers_of_eval_points_for_chunks[1]),
-        ];
+        let evals = ConsecutiveEvals {
+            zeta: self
+                .evals
+                .zeta
+                .combine(powers_of_eval_points_for_chunks[Z_IDX]),
```

Comment:
might want to fix powers_of_eval_points_for_chunks to be a type as well, instead of an array :p

Reply:
Good point